junifer/markers/parcel_aggregation.py

```diff
@@ -11,7 +11,7 @@
 from nilearn.maskers import NiftiMasker
```

Comment:
same here: in the ParcelAggregation init method should masks not be this should be str or list of (str or dict) now?

Reply:
same as before. This is for computing masks.